### PR TITLE
feat(netflix): Improve Availability Context Dropdown

### PR DESCRIPTION
**Pull request by jrsquared**

### CODE REVIEW

File: app/scripts/modules/netflix/availability/availability.trend.component.ts [hunk 1]
```diff
@@ -51,11 +57,18 @@ export class AvailabilityTrendController implements ng.IComponentController {
 
     this.availabilityWindow.ts.is_outage.forEach((isOutage, index) => {
       if (isOutage) {
+        const popoverContent: PopoverContent = {
+          datetime: get(this.availabilityWindow, ['ts', 'datetime', index],  'Unknown'),
+          availability: get(this.availabilityWindow, ['ts', 'availability', index], 0),
+          incidents: get(this.availabilityWindow, ['ts', 'incs', index], <string[]>[])
```
Comment (icfantv) [inline app/scripts/modules/netflix/availability/availability.trend.component.ts]: is this cast needed?  what happens if you just do `string[]`?

File: app/scripts/modules/netflix/availability/availability.trend.component.ts [hunk 2]
```diff
@@ -102,7 +115,7 @@ export class AvailabilityTrendController implements ng.IComponentController {
         () => {
           this.popoverOpen[index] = false;
         },
-        300);
+        500);
```
Comment (icfantv) [inline app/scripts/modules/netflix/availability/availability.trend.component.ts]: 300 was SO last week.

File: app/scripts/modules/netflix/help/netflixHelpContents.registry.js [hunk 3]
```diff
@@ -74,6 +74,10 @@ module.exports = angular
                     <li>none</li>
                   </ul>
                   <a href="https://confluence.netflix.com/display/PDCLOUD/Validation+for+Persisted+Properties" target="_blank">Full Documentation</a>`
+      },
+      {
+        key: 'availability.context',
+        contents: 'The availability trends for the Netflix service represented as \'nines\' and availability percentage. <a href="http://go/availabilitycontextdoc" target="_blank">Click here</a> for more details.'
```
Comment (icfantv) [inline app/scripts/modules/netflix/help/netflixHelpContents.registry.js]: I feel like this isn't a complete sentence and I'm left wanting for more.

File: app/scripts/modules/netflix/availability/availability.less [hunk 4]
```diff
@@ -149,6 +152,9 @@
       padding: 15px 20px;
       color: #fff;
     }
+    .glyphicon {
```
Comment (icfantv) [inline app/scripts/modules/netflix/availability/availability.less]: I missed this the first time around.  We should be moving to font-awesome.  Apologies.

File: app/scripts/modules/netflix/availability/availability.trend.component.ts (hunk 1 above)
Comment (jrsquared) [inline app/scripts/modules/netflix/availability/availability.trend.component.ts]: Yes. Complains about implicit 'any'

File: app/scripts/modules/netflix/availability/availability.less (hunk 4 above)
Comment (jrsquared) [inline app/scripts/modules/netflix/availability/availability.less]: Outside the scope of this PR. The glyph icon comes from help-field, so should be a part of converting help-field.

File: app/scripts/modules/netflix/help/netflixHelpContents.registry.js (hunk 3 above)
Comment (jrsquared) [inline app/scripts/modules/netflix/help/netflixHelpContents.registry.js]: Yep, agree. Will fix before merge.

File: app/scripts/modules/netflix/availability/availability.trend.component.ts (hunk 2 above)
Comment (jrsquared) [inline app/scripts/modules/netflix/availability/availability.trend.component.ts]: Heh, for context the tooltips are small enough that 300 felt like not enough time to get the cursor to the popup.

File: app/scripts/modules/netflix/availability/availability.less (hunk 4 above)
Comment (icfantv) [inline app/scripts/modules/netflix/availability/availability.less]: oooooh.  ok.

**Pull request merged**